src/components/Navbar.jsx

```diff
@@ -1,12 +1,27 @@
 // LIBRARY IMPORTS
 import React from 'react';
+import { NavLink } from 'react-router-dom';
 
 // LOCAL IMPORTS
 
 export default function NavBar() {
   return (
-    <div>
-      Navigation Bar will go here.
+    <div className="space-x-4">
+      <NavLink
```

Comment:
NavLink is a React Router component that allows for more styling than React router's Link component - especially around active vs inactive link components.